modules/signals/entities/spec/updaters/prepend-entity.spec.ts

```diff
@@ -0,0 +1,215 @@
+import { patchState, signalStore, type } from '@ngrx/signals';
+import {
+  prependEntity,
+  entityConfig,
+  withEntities,
+  addEntity,
+} from '../../src';
+import { Todo, todo1, todo2, todo3, User, user1, user2 } from '../mocks';
```

Comment:
This is not used and can be removed:

```suggestion
import { Todo, todo1, todo2, User, user1, user2 } from '../mocks';
```